tests/tests_pytorch/callbacks/test_lr_monitor.py

```diff
@@ -626,3 +626,39 @@ def configure_optimizers(self):
     assert list(lr_monitor.last_momentum_values) == ["lr-Adam/pg1-momentum", "lr-Adam/pg2-momentum"]
     assert all(val == momentum for val in lr_monitor.last_momentum_values.values())
     assert all(all(val == lr for val in lr_monitor.lrs[lr_key]) for lr_key in lr_monitor.lrs)
+
+
+def test_lr_monitor_early_stopping(tmpdir):
```

Comment:
```suggestion
def test_lr_monitor_updates_callback_metrics(tmpdir):
```
The feature is not specific to early stopping. Early stopping here is only used as an end-to-end vehicle for testing. Let's update the description too to make it clear. Otherwise, there is no indication in this test to what part of the LearningRateMonitor this relates to.